### PR TITLE
Added onclick function binding to chart nodes

### PR DESCRIPTION
**Pull request by ndrqu**
Now you can bind functions to your chart nodes by writing `:$myFunction`

### CODE REVIEW

**Comment by adrai** [comment]
Can you add a bit more documentation in the Readme?

**Comment by ndrqu** [comment]
Hi, I don't see much documentation in the Readme. I just see some inline comments in the sample code. Do you want me to put it there?

**Comment by adrai** [comment]
Yes, just to see what myFunction could be...

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
+    "nodemon": "^1.19.0",
```
Comment (adrai) [inline package.json]: Is there a reason for this dependency?

Comment (ndrqu) [reply package.json]: No, it's just for me to run the application.

I am sorry for this, I'm new to GitHub. Should I remove this?
I also made a mistake with a closing bracket and I'm trying to to update with the fix

Comment (adrai) [reply package.json]: Yes, remove it

Comment (ndrqu) [reply package.json]: done

**Comment by adrai** [comment]
landed in v1.12.0

**Pull request merged**